.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = libaditof
 	url = https://github.com/analogdevicesinc/libaditof.git
 	branch = main
+[submodule "bindings/python/pybind11"]
+	path = bindings/python/pybind11
+	url = https://github.com/pybind/pybind11
```

Comment:
Please specify a stable release version (probably  v2.11.2). Using main version may cause different issues.

Reply:
👍. Pinned `--remote` updates to the v2.11 branch.